demosplan/DemosPlanCoreBundle/Logic/Statement/StatementGeoService.php

```diff
@@ -602,9 +603,9 @@ protected function getDataFromPoints($points, $geoResults, $tempStatementId)
      * This is in StatementService, since it's only used there. It's not really about the procedure, it's actually
      * about the statement.
      */
-    private function isStatementOfProcedurePartOfWind(int $windNumber, string $procedureId): bool
+    private function isStatementOfProcedurePartOfWind(int $windNumber, string $procedureId, DatasheetService $datasheetService): bool
```

Comment:
You need to inject this as dependency or pass it on calling

Reply:
thx, done in https://github.com/demos-europe/demosplan-core/pull/1408/commits/430c2f983f76c58dd7d8e0942923d8631aa82c2b